docs/agent/secrets.md

```diff
@@ -191,7 +191,7 @@ will refuse to use it otherwise):
 - Have at least `exec` right for the owner.
 
 Also:
-- The executable will not share any environment variables with the agent.
+- The executable will share the same environment variables than agent.
```

Comment:
```suggestion
- The executable will share the same environment variables as the agent.
```

Reply:
Okay, but what are the security implications, and why are we now doing this?

Reply:
The env is needed on windows. Many software (included python for example) need the environment to initialize things like random number generator.

Also this would mean sharing the environment of the agent user so the only sensitive information will most likely be the `API_KEY`. This information is available to the executable in any case: we exec the user software with the same user as the agent, which mean that executable can read the API token from the disk and query the agent configuration anyway. 

@mtremsal and @derekwbrown might have more input too.

Reply:
Makes sense to me, we should just clarify

Cc @pratikgs-dd @jmassey-datadog

Reply:
Ping @trishankatdatadog @mtremsal. Can we merge this ?